README.md

```diff
@@ -0,0 +1,52 @@
+# json_transport
+
+Providing schemaless transport over ROS pub/sub via JSON. This is useful when schema is enforced elsewhere, or the data is truly schemaless (i.e. parameters, diagnostics).
+
+The [`nlohmann/json.hpp`](https://github.com/nlohmann/json/blob/develop/single_include/nlohmann/json.hpp) library is included directly, but can be dropped for an apt dependency in artful, or if the package is bloomed into the rosdistro.
+
+## C++
+
+The provided adapters allows publishing and subscribing [`nlohmann::json`](https://github.com/nlohmann/json) datatypes.
```

Comment:
Because this is public facing I'll be pedantic. Feel free to ignore.

`provided adapters allows` should be `adapter allows` or `adapters allow`